### PR TITLE
Add base Flutter dependencies

### PR DESCRIPTION
**Pull request by abelfodil**
- Storage: https://pub.dev/packages/hive
- OTG: https://pub.dev/packages/usb_serial
- Bluetooth: https://pub.dev/packages/flutter_blue
- BLoC: https://pub.dev/packages/flutter_bloc
- Query battery: https://pub.dev/packages/battery
- Export data: https://pub.dev/packages/share
- Zip: https://pub.dev/packages/archive

### CODE REVIEW

**Review by MouradLachhab — APPROVED** [review]

**Comment by WilliamHarvey97** [comment]
https://pub.dev/packages/hive pourrait être intéressant.

File: mobile/pubspec.yaml [hunk 1]
```diff
@@ -28,6 +28,13 @@ dependencies:
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^0.1.3
   protobuf: ^1.0.1
+  flutter_secure_storage: ^3.3.3
```
Comment (WilliamHarvey97) [inline mobile/pubspec.yaml:31]: https://pub.dev/packages/hive pourrait être intéressant.

File: mobile/pubspec.yaml [hunk 2]
```diff
@@ -28,6 +28,13 @@ dependencies:
...
+  usb_serial: ^0.2.4
+  flutter_blue: ^0.7.2
+  flutter_bloc: ^6.0.5
```
Comment (WilliamHarvey97) [inline mobile/pubspec.yaml:34]: Si on utilise flutter_bloc, il faut aussi que notre app utilise le code de (bloc)[https://pub.dev/packages/bloc]

**Comment by abelfodil** [comment]
@WilliamHarvey97 everything gucci?

**Review by conorato — APPROVED** [review]

**Comment by WilliamHarvey97** [comment]
> @WilliamHarvey97 everything gucci?

Parfait!

**Review by WilliamHarvey97 — APPROVED** [review]

**Pull request merged**